hudi-common/src/main/java/org/apache/hudi/common/model/HoodiePreCombineAvroRecordMerger.java

```diff
@@ -35,8 +35,16 @@ public class HoodiePreCombineAvroRecordMerger extends HoodieAvroRecordMerger {
   public static final HoodiePreCombineAvroRecordMerger INSTANCE = new HoodiePreCombineAvroRecordMerger();
```

Comment:
I assume this merger will be removed once the merging stage (e.g., `PRE_COMBINE`, `COMBINE`) is added to the merge API.

Reply:
Yeah, since we don't need it.